src/main/java/com/liferay/faces/util/el/ELResolverBase.java

```diff
@@ -37,13 +42,38 @@
  */
 public abstract class ELResolverBase extends ELResolver implements Serializable {
 
+	// Logger
+	private static final Logger logger = LoggerFactory.getLogger(ELResolverBase.class);
+
 	// serialVersionUID
 	private static final long serialVersionUID = 8075201303544048292L;
 
 	// Private Constants
-	private static final ArrayList<FeatureDescriptor> FEATURE_DESCRIPTORS = new ArrayList<FeatureDescriptor>();
+	private final List<FeatureDescriptor> featureDescriptors;
```

Comment:
@stiemannkj1 As I recall, usage of the final keyword for private data members is not commonly used in the codebase.